db/migrate/20180815180513_add_required_to_user_name.rb

```diff
@@ -0,0 +1,6 @@
+class AddRequiredToUserName < ActiveRecord::Migration[5.1]
+  def change
+    change_column_null :users, :first_name, false
```

Comment:
I like this approach, but...  knowing that this the migration where you are creating this column is in this same PR i would recommend to update the migration where you  added them for the first time. the `AddNameToUser` one. you need to make a rollback to that migration, then update it, and then run it again.